src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -155,6 +155,7 @@ public void writeThrough(User user) {
     userEntity.setProperty("admin",user.isAdmin());
```

Comment:
The writeThrough function is only called when a new user is registered, so this won't save the profile content.